package.json

```
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --port ${PORT:-8080}",
```

Can you change the default to 5173, that's vite's default

```suggestion
    "dev": "vite --port ${PORT:-5173}",
```